Source/WebCore/platform/audio/gstreamer/AudioFileReaderGStreamer.cpp

```diff
@@ -253,7 +253,7 @@ GstFlowReturn AudioFileReader::handleSample(GstAppSink* sink)
 
 void AudioFileReader::handleMessage(GstMessage* message)
 {
-    ASSERT(&m_runLoop == &RunLoop::current());
+    ASSERT(m_runLoop.isCurrent());
```

Comment:
i think now it's better to use:
   assertIsCurrent(m_runLoop);
So you get the thread safety analysis should the future variable declarations start using it.